client/app/bundles/shared/components/Resource/Resources.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import css from './Resources.scss';
+import Tag from '../Tag';
+
+type Props = {
+  resourcesObj?: Object,
+  tagged?: boolean,
+};
+
+export default class Resources extends React.Component<Props> {
```

Comment:
Oooh, sorry I didn't catch this before, but can we rename this to `Resource`? Just singular, not plural.